docs/en/integrations/data-ingestion/data-formats/parquet-arrow-avro-orc.md

```diff
@@ -0,0 +1,325 @@
+
+Let's create a table to import parquet data to:
+
+```sql
+CREATE TABLE sometable
+(
+    `path` String,
+    `date` Date,
+    `hits` UInt32
+)
+ENGINE = MergeTree
+ORDER BY (date, path)
+```
+
+Now we can import data using a `FROM INFILE` clause:
+
+
+```sql
+INSERT INTO sometable
+FROM INFILE 'data.parquet' FORMAT Parquet;
+
+SELECT *
+FROM sometable
+LIMIT 5;
+```
+```response
+┌─path──────────────────────────┬───────date─┬─hits─┐
+│ 1988_in_philosophy            │ 2015-05-01 │   70 │
+│ 2004_Green_Bay_Packers_season │ 2015-05-01 │  970 │
+│ 24_hours_of_lemans            │ 2015-05-01 │   37 │
+│ 25604_Karlin                  │ 2015-05-01 │   20 │
+│ ASCII_ART                     │ 2015-05-01 │    9 │
+└───────────────────────────────┴────────────┴──────┘
+```
+
+Note how ClickHouse automatically converted parquet strings (in the `date` column) to the `Date` type. This is because ClickHouse does a typecast automatically based on the types in the target table.
+
+
+### Creating new tables from parquet files
+
+Since ClickHouse reads parquet file schema, we can create tables on the fly:
+
+```sql
+CREATE TABLE imported_from_parquet
+ENGINE = MergeTree
+ORDER BY tuple() AS
+SELECT *
+FROM file('data.parquet', Parquet)
+```
+
+This will automatically create and populate a table from a given parquet file:
+
+```sql
+DESCRIBE TABLE imported_from_parquet;
+```
+```response
+┌─name─┬─type─────────────┬─default_type─┬─default_expression─┬─comment─┬─codec_expression─┬─ttl_expression─┐
+│ path │ Nullable(String) │              │                    │         │                  │                │
+│ date │ Nullable(String) │              │                    │         │                  │                │
+│ hits │ Nullable(Int64)  │              │                    │         │                  │                │
+└──────┴──────────────────┴──────────────┴────────────────────┴─────────┴──────────────────┴────────────────┘
+```
+
+By default, ClickHouse is strict with column names, types, and values. But sometimes, we can skip unexistent columns or unsupported values during import. This can be managed with [Parquet settings](/docs/en/interfaces/formats.md/#parquet-format-settings).
+
+
+### Exporting to Parquet format
+
+:::tip
+When using `INTO OUTFILE` with ClickHouse Cloud you will need to run the commands in `clickhouse client` on the machine where the file will be written to.
+:::
+
+To export any table or query result to the Parquet file, we can use an `INTO OUTFILE` clause:
+
+```sql
+SELECT *
+FROM sometable
+INTO OUTFILE 'export.parquet'
+FORMAT Parquet
+```
+
+This will create the `export.parquet` file in a working directory.
+
+### ClickHouse and Parquet data types
+ClickHouse and Parquet data types are mostly identical but still [differ a bit](/docs/en/interfaces/formats.md/#data-types-matching-parquet). For example, ClickHouse will import Parquets' `TIME` type as UInt32 by default ([time.parquet file](assets/time.parquet)):
```

Comment:
That's not true. Parquet `TIME` type will be converted to ClickHouse `DataType64` type with according scale. And specified sample file doesn't contain columns with `TIME` type:
```
avogar@avogar-dev:~/tmp/data$ parquet-tools schema time.parquet
message schema {
  required int64 n (UINT_64);
  required int64 time;
}
```

It contains column `time` with `int64` type and it will be inferred as ClickHouse  `Int64`.
To create a file with type `TIME` you can use the next python script:
```py
import pyarrow as pa
import pyarrow.parquet as pq

import time
time = pa.array([time.time() * 1000], pa.time64('us'))
table  = pa.table([time], ["time"])
pq.write_table(table, "test.parquet", version=')
```

It will create a file with the next schema:
```
avogar@avogar-dev:~/tmp/data$ parquet-tools schema test.parquet
message schema {
  optional int64 time (TIME_MICROS);
}
```

Reply:
Good hit. That's because I first exported ClickHouse' `DateTime` to Parquet and got this:
```sql
CREATE TABLE test (`time` DateTime ) ENGINE = Memory;

SELECT * FROM test INTO OUTFILE '/var/lib/clickhouse/user_files/time.parquet' FORMAT Parquet;

┌─name─┬─type────────────┬─default_type─┬─default_expression─┬─comment─┬─codec_expression─┬─ttl_expression─┐
│ time │ Nullable(Int64) │              │                    │         │                  │                │
└──────┴─────────────────┴──────────────┴────────────────────┴─────────┴──────────────────┴────────────────┘
```

I've changed wording not to confuse readers, but the idea of the section was to mention type conversion is an easy fix for types misinterpretation. Created PR: https://github.com/ClickHouse/clickhouse-docs/pull/711